DataConnectors/CiscoUmbrella/CiscoUmbrella_API_FunctionApp.json

```diff
@@ -0,0 +1,171 @@
+            "instructions":[
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Workspace ID"
+                    },
+                    "type": "CopyableLabel"
+                },
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "PrimaryKey"
+                        ],
+                        "label": "Primary Key"
+                    },
+                    "type": "CopyableLabel"
+                }
+            ]
+        },
+        {
+            "title": "Option 1 - Azure Resource Manager (ARM) Template",
+            "description": "Use this method for automated deployment of the Cisco Umbrella data connector using an ARM Tempate.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/sentinelciscoumbrellaazuredeploy)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n3. Enter the **Workspace ID**, **Workspace Key**, **S3Bucket**, **AWSAccessKeyId**, **AWSSecretAccesKey**, **AWSRegionName** and deploy."
+        },
+        {
+            "title": "Option 2 - Manual Deployment of Azure Functions (Deploy via VS Code)",
+            "description": "Use the following step-by-step instructions to deploy the Cisco Umbrella connector manually with Azure Functions."
+        },
+        {
+            "title": "",
+            "description": "**1. Deploy a Function App via VS Code**\n\n> NOTE: You will need to prepare VS code for Azure function development. See [link](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-powershell#prerequisites)\n\n1. Download the [Zip file](https://github.com/Azure/Azure-Sentinel/blob/CiscoUmbrellaConnector/DataConnectors/CiscoUmbrella/CiscoUmbrellaConn.zip?raw=true) of the Azure Funciton app from Github.\n2. Extract to a location on your local host.\n3. Open VS Code.\n4. Click File -> Open Folder.\n5. Select the top level folder from extracted files.\n6. Click Azure Functions: Deploy to function app. You maybe asked to sign in to Azure.\n7. Click Create New function app in Azure (advanced).\n8. Provide a unique name like \"Umbrellaxxx\". Press Enter.\n9. Select runtime stack: Python 3.8.\n10. Hosting plan: click Consumption.\n11. Click Create new Resource Group or select existing.\n12. Click Create a new storage Account or select existing.\n13. Click Create new Application Insights resource.\n14. Pick a location to deploy in.\n15. Deployment will begin.\n16. Wait for the deployment to complete.\n17. Go to the Azure Portal."
```

Comment:
The link in this step points to PowerShell pre-requisites - https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-powershell#prerequisites 

Please point to Python pre-requisites as this Azure Function is Python based.

Reply:
done